src/mount/mount-tool.c

```diff
@@ -607,6 +620,153 @@ static int start_transient_automount(
         return 0;
 }
 
+static int stop_transient_mount(
+                sd_bus *bus,
```

Comment:
This code should actually work for all kinds of mount units not just transient ones. I hence would rename it to just stop_mount()